settings.py

```diff
@@ -304,6 +300,8 @@
 SEQR_TOS_VERSION = float(os.environ.get('SEQR_TOS_VERSION', 1.2))
 
 BASE_URL = os.environ.get("BASE_URL", "/")
+if DEPLOYMENT_TYPE in {'prod', 'dev'}:
```

Comment:
probably neater to move the BASE_URL definition up higher in the file so we can use the one conditional block for this